src/backend/distributed/Makefile

```diff
@@ -119,6 +119,8 @@ $(EXTENSION)--6.1-11.sql: $(EXTENSION)--6.1-10.sql $(EXTENSION)--6.1-10--6.1-11.
 	cat $^ > $@
 $(EXTENSION)--6.1-12.sql: $(EXTENSION)--6.1-11.sql $(EXTENSION)--6.1-11--6.1-12.sql
 	cat $^ > $@
+$(EXTENSION)--6.1-13.sql: $(EXTENSION)--6.1-12.sql $(EXTENSION)--6.1-12--6.1-13.sql
```

Comment:
(a) I think it's about time we do something about these numbers (maybe something more automatic? No clue, but the directory listing is getting needlessly long in my Eclipse window).

(b) Shouldn't we have changed to `6.2` as soon as `6.1` was released?

/cc @anarazel for ideas/commentary on both of the above points.